symExec.py

```diff
@@ -1740,5 +1727,16 @@ def print_state(block_address, stack, mem, global_state):
     if PRINT_MODE: print "MEM: " + str(mem)
     if PRINT_MODE: print "GLOBAL STATE: " + str(global_state)
 
+def contains_only_concrete_values(stack):
+    for element in stack:
+        if not isinstance(element, (int, long)):
```

Comment:
checkout this helper function - https://github.com/melonproject/oyente/blob/master/symExec.py#L80